app/src/main/java/uc/edu/klopfsea/groupscheduler/ui/main/MainViewModel.kt

```diff
@@ -8,7 +8,7 @@ import com.google.firebase.firestore.FirebaseFirestoreSettings
  * connection and build the firebase firestore into our code
  */
 class MainViewModel : ViewModel() {
-    private var firestore: FirebaseFirestore = FirebaseFirestore.getInstance()
+    public var firestore: FirebaseFirestore = FirebaseFirestore.getInstance()
 
     init {
```

Comment:
I felt making this public would make it easier for debugging purposes

Reply:
Not necessarily.  That opens it up so that it can be called outside of this program.... which is a bit dangerous, since Firebase is for persistence.